homeassistant/components/adguard/services.yaml

```diff
@@ -1,6 +1,6 @@
 add_url:
   name: Add url
-  description: Add a new filter subscription to AdGuard Home.
+  description: Add a new allowlist/blocklist filter subscription to AdGuard Home.
```

Comment:
Since 2023.8 the name and description of services have moved to strings.json, please rebase and move them and move the strings